AK/Debug.h.in

```diff
@@ -1,19 +1,20 @@
 /*
  * Copyright (c) 2020, the SerenityOS developers.
+ * Copyright (c) 2022, the SerenityOS developers.
```

Comment:
You can just extend the year (unless you meant to add your own name)

Reply:
Done.